### PR TITLE
v1.21.0-rc

### PR DESCRIPTION
**Pull request by SaitejaTamma**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind release 

**What this PR does / Why we need it**:
v1.20.0-rc
**Which issue(s) this PR fixes**:
#2469

**Special notes for your reviewer**:

### CODE REVIEW

File: sdks/nodejs/package-lock.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< Updated upstream
```
Comment (markmandel) [inline sdks/nodejs/package-lock.json:2]: This looks wrong 😬 not sure what happened here?

File: site/content/en/blog/releases/1.21.0-rc.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: "1.21.0 - Release Candidate"
+linkTitle: "1.21.0-rc"
+date: "2022-02-08"
+---
+This is the 1.21.0-rc release of Agones.
+
+This release brings us the version of node 16 and npm 7
```
Comment (markmandel) [inline site/content/en/blog/releases/1.21.0-rc.md:8]:
```suggestion
This is a big bug fix release!

Not only did we fix a tonne of flaky tests (which will make contributing easier), but also solved lots of tricky bugs in the process. This includes fixes to the alpha Fleet Autoscaling custom syncing, as well as quality of life improvements where we now validate `metadata` patches on `GameServerAllocation`.

If you are using Agones' metrics, note that there is **breaking change**, in that the `node_name` label has been removed from allocation metrics to remove a potential cardinality bomb.
```

WDYT?

**Review by markmandel — APPROVED** [review]
LGTM 👍🏻 

Please feel free to merge when you are ready to move on to the next steps. (I think you can? If not, ping me and I will push the button).

File: sdks/nodejs/package-lock.json [hunk 3]
```diff
@@ -2514,6 +2514,51 @@
             "engines": {
                 "node": ">=10"
             }
+=======
```
Comment (roberthbailey) [inline sdks/nodejs/package-lock.json:2517]: This file looks like it has merge conflicts in it. Please make sure you are using nodejs 16 when running the command (or alternately just edit the version line by hand).

See https://github.com/googleforgames/agones/pull/2465

File: site/content/en/blog/releases/1.21.0-rc.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+title: "1.21.0 - Release Candidate"
+linkTitle: "1.21.0-rc"
+date: "2022-02-08"
+---
+This is the 1.21.0-rc release of Agones.
+
+This is a big bug fix release!
+
+Not only did we fix a tonne of flaky tests (which will make contributing easier), but also solved lots of tricky bugs in the process. This includes fixes to the alpha Fleet Autoscaling custom syncing, as well as quality of life improvements where we now validate `metadata` patches on `GameServerAllocation`.
```
Comment (roberthbailey) [inline site/content/en/blog/releases/1.21.0-rc.md:10]: nit: s/tonne/ton

**Review by roberthbailey — COMMENTED** [review]
LGTM except for the nodejs package-lock.json file, which needs to be fixed before this PR gets merged.

File: site/content/en/blog/releases/1.21.0-rc.md (hunk 4 above)
Comment (steven-supersolid) [reply site/content/en/blog/releases/1.21.0-rc.md:10]: Is it an imperial ton or metric ton (tonne) though, and in the latter case should we use the SI unit of megagrams instead? :P

File: sdks/nodejs/package-lock.json (hunk 3 above)
Comment (markmandel) [reply sdks/nodejs/package-lock.json:2517]: oh shoot, I thought this was fixed last night. Thanks for picking that up, I missed it late at night.

File: site/content/en/blog/releases/1.21.0-rc.md (hunk 4 above)
Comment (markmandel) [reply site/content/en/blog/releases/1.21.0-rc.md:10]: https://en.wikipedia.org/wiki/Tonne 😄

**Comment by markmandel** [comment]
That looks better 👍🏻

**Comment by steven-supersolid** [comment]
Double checked the package-lock.json and looks good

**Pull request merged**